packages/playwright/src/runner/runner.ts

```diff
@@ -152,6 +152,7 @@ export class Runner {
 export type LastRunInfo = {
   status: FullResult['status'];
   failedTests: string[];
+  testDurations?: { [testId: string]: number };
```

Comment:
We are a bit worried that this will force `testId` to never change to preserve backwards compatibility. Therefore, we would like to use `testCase.titlePath()` instead of id for now. Something like this:

```ts
export type LastRunInfo = {
  status: FullResult['status'],
  projects: {
    name: string,
    files: {
      path: string,
      tests: {
        title: string[],
        duration: number,
        status: TestStatus,
      }[],
    }[],
  }[],
};
```
 
Here `file.path` is `toPosixPath(path.relative(config.rootDir, filePath))`, similar to [this line](https://github.com/microsoft/playwright/blob/main/packages/playwright/src/common/suiteUtils.ts#L56).

Reply:
I just tried to write the LastRunInfo in the proposed format, but its not as straight forward as I thought it would be.

> That said, you can leave it as is, and we'll iterate on it.

You mean, I don't need to proceed here to get this merged? 🙏

Reply:
> […] we do not expect last-run.json to be used between different versions

In our CI setup, I'm using pipeline steps to restore the last-run.json from the last successful main build. So, in our case last-run info from previous playwright versions will be used, and it would be great if there is some level of backward compatibility... at least it should not fail when the format is incorrect.